python/ray/serve/tests/test_air_integrations.py

```diff
@@ -210,20 +210,17 @@ def test_air_integrations_in_pipeline(serve_instance):
 
 
 def test_air_integrations_reconfigure(serve_instance):
-    path = tempfile.mkdtemp()
-    uri = f"file://{path}/test_uri"
-    Checkpoint.from_dict({"increment": 2}).to_uri(uri)
+    checkpoint = Checkpoint.from_dict({"increment": 2})
 
     predictor_cls = "ray.serve.tests.test_air_integrations.AdderPredictor"
     additional_config = {
-        "checkpoint": {"increment": 5},
-        "predictor_cls": "ray.serve.tests.test_air_integrations.AdderPredictor",
+        "checkpoint": Checkpoint.from_dict({"increment": 5}),
```

Comment:
Shouldn't we have to keep the checkpoint as a URI here ?
([LINK](https://github.com/ray-project/ray/blob/ea94cda1f3d0dc7fd39e259c7e82a07bdc9fafd3/python/ray/air/checkpoint.py#L117))